cohere/client.py

```diff
@@ -224,14 +224,32 @@ def detect_language(self, texts: List[str]) -> List[Language]:
             results.append(Language(result["language_code"], result["language_name"]))
         return DetectLanguageResponse(results)
 
-    def feedback(self, id: str, feedback: str, accepted: bool):
+    def feedback(self, id: str, good_generation: bool, feedback: str = "") -> Feedback:
```

Comment:
I don't think we need this function in the client. It's specific to Generate, so just having it in the Generation class is a better place

Reply:
I can definitely move this under generate, but I thought that might 1) hurt discoverability, and 2) we might want to support feedback for other endpoints in the future. Thoughts?